docs/installations/installation-on-gnu-linux.md

```diff
@@ -0,0 +1,106 @@
+Installation on GNU/Linux
+=========================
+
+## Install via Conda
+
+Use conda or bioconda to install OpenMS.
+
+1. Follow the instructions to [install conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/macos.html).
```

Comment:
the Linux OpenMS installation instructions link to the macOS conda installation instructions.
this should be the correct link: https://docs.conda.io/projects/conda/en/latest/user-guide/install/linux.html